### PR TITLE
Fixes russian red, also changes oxy name

### PR DESCRIPTION
**Pull request by Alterist**
broken by #615

### CODE REVIEW

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -309,15 +303,14 @@
...
 /obj/item/storage/pill_bottle/happy
-	name = "\improper Happy pill bottle"
+	name = "\improper happy pill bottle"
 	desc = "Contains highly illegal drugs. When you want to see the rainbow."
 	max_storage_space = 7
 	pill_type_to_fill = /obj/item/reagent_container/pill/happy
 
 /obj/item/storage/pill_bottle/zoom
-	name = "\improper Zoom pill bottle"
+	name = "\improper zoom pill bottle"
```
Comment (Ghommie) [inline code/game/objects/items/storage/firstaid.dm]: remove the `\improper` then.

File: code/game/objects/items/storage/firstaid.dm [hunk 2]
```diff
@@ -309,15 +303,14 @@
 	icon_state = "pill_canister9"
 	pill_type_to_fill = /obj/item/reagent_container/pill/tricordrazine
 
-
 /obj/item/storage/pill_bottle/happy
-	name = "\improper Happy pill bottle"
+	name = "\improper happy pill bottle"
```
Comment (Ghommie) [inline code/game/objects/items/storage/firstaid.dm]: same.

File: code/game/objects/items/storage/firstaid.dm (hunk 1 above)
Comment (Alterist) [reply code/game/objects/items/storage/firstaid.dm]: Done

File: code/game/objects/items/storage/firstaid.dm (hunk 2 above)
Comment (Alterist) [reply code/game/objects/items/storage/firstaid.dm]: Done

**Pull request merged**